docs/content/developers/ontwerpkeuzes-informatiemodellen.md

```diff
@@ -1,2 +1,15 @@
-Bij het ontwikkelen van de API's aam de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie hier) als consequentie hebbe dat een informatiemodel aangpast moet worden. 
+## Aanpassingen RGBZ en ImZTC
+
+Bij het ontwikkelen van de API's aan de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie [hier](https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/content/developers/design-keuzes.md)) als consequentie hebben dat een informatiemodel aangepast moet worden. 
 Hieronder vermelden we de gemaakte keuzes tot aanpassing van RGBZ en ImZTC. 
+
+[nog uit te werken:]
+- Einddatum van zaak als afgeleid gegeven (en STATUSTYPE in ImZTC met hoogste volgnummer is laatste status). 
+- Omschrijving- en Toelichting-attributen niet meer AN1000 maar AN ('string'); ook BESLUIT.Publicatietekst
+- Richting van relatie ZAAK - INFORMATIEOBJECT omdraaien
+- Richting van relatie BESLUIT - INFORMATIEOBJECT omdraaien 
+- Formele historie op relaties van BESLUIT: ze kunnen gewijzigd worden
```

Comment:
Betekent dit dat je ook moet `registratiedatum` van IO bij besluit vastleggen?

Reply:
@sergei-maertens  Dat is dan weer niet nodig. 
Oplossen met generieke oplossing voor formele en materiele historie.

Reply:
Net het vastleggen van die datum is toch onderdeel daarvan? Je kan het niet op besluit zelf of informatieobject bijhouden door het m-op-n karakter. Wil ik het een keer face to face over hebben, zodat we niet naast elkaar praten :smile: